tests/baselines/reference/genericRestParameters1.errors.txt

```diff
@@ -26,7 +26,7 @@ tests/cases/conformance/types/rest/genericRestParameters1.ts(31,1): error TS2556
     f1(ns[0], ns[1], true);
     f1(...ns, true);  // Error, tuple spread only expanded when last
     ~~~~~~~~~~~~~~~
-!!! error TS2556: Expected 3 arguments, but got 1 or more.
+!!! error TS2557: Expected at least 3 arguments, but got 1 or more.

```

Comment:
These still emit a slightly changed error message? Is the changed behavior in this case wanted and/or acceptable?